addons/account/models/account_move.py

```diff
@@ -3535,6 +3535,11 @@ def _query_get(self, domain=None):
         if not isinstance(domain, (list, tuple)):
             domain = safe_eval(domain)
 
+        domain += [
+            ('display_type', 'not in', ('line_section', 'line_note')),
+            ('move_id.state', '!=', 'cancel'),
+        ]
+
```

Comment:
Weird, it is already there below (and even then I think it is weird to override this so extensively, seems like a rather low-level method)